packages/tables/frontend/sirius-components-tables/src/graphql/mutation/useTableMutation.types.ts

```diff
@@ -58,3 +59,22 @@ export interface GQLChangeColumnVisibilityData {
 }
 
 export type GQLChangeTableColumnVisibilityPayload = GQLErrorPayload | GQLSuccessPayload;
+
+export interface GQLResizeRowInput {
+  id: string;
+  editingContextId: string;
+  representationId: string;
+  tableId: string;
+  rowId: string;
+  height: number;
```

Comment:
I'm ok with this but it's fun to notice that it's not nullable on the frontend side...